From 9c4b9e9268d3a22cc75ec56a25d409760f55ef70 Mon Sep 17 00:00:00 2001
From: MSe1969 <mse1969@posteo.de>
Date: Mon, 15 Nov 2021 18:45:53 +0100
Subject: [PATCH] Exclude recovery as A/B OTA partition

As I use a special version of the Lineage recovery for the hardened build,
I would like to avoid, that the recovery always gets overwritten, when
flashing LineageOS.

Credits to: stay steadfast <steadfasterX@users.noreply.github.com>

Change-Id: I8d4a6435576b3a1503e455bd696720ac1bb700aa
---
 BoardConfig.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 86c70a1..a12bc99 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -13,7 +13,6 @@ DEVICE_PATH := device/oneplus/hotdog
 AB_OTA_PARTITIONS += \
     odm \
     product \
-    recovery \
     system_ext \
     vbmeta_system
 
-- 
2.30.2

